app/assets/javascripts/models/beta/iteration.js

```diff
@@ -13,11 +13,25 @@ const getIterationForDate = (date, project) => {
     : iterationQuantity + 1;
 };
 
+const getNextIterationNumber = (sprints) => {
+  const lastSprint = sprints[sprints.length - 1];
```

Comment:
I've checked that this repository has [underscorejs](https://underscorejs.org/#last) so, you could use the `last` function here.
```js
import { last } from 'underscore';
...
const lastSprint = last(sprints);

Reply:
uuu, fancy :smile: